packages/openssh/build.sh

```diff
@@ -28,6 +28,7 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 --with-pid-dir=$TERMUX_PREFIX/var/run
 --with-privsep-path=$TERMUX_PREFIX/var/empty
 --with-xauth=$TERMUX_PREFIX/bin/xauth
+--with-kerberos5=$TERMUX_PREFIX/usr
 ac_cv_func_endgrent=yes
```

Comment:
`$TERMUX_PREFIX`already includes `/usr`

Reply:
Sorry, my mistake. I tested it and it worked, so I didn't notice.
Fixed.